whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/AsBlockSpec.groovy

```diff
@@ -513,7 +513,7 @@ class AsBlockSpec extends BaseQueryUpdateSpec {
     def "modify as-block, add multiple mnt-by"() {
 
         given:
-        syncUpdate(getTransient("RIPE-DBM-STARTUP-MNT") + "password:startup")
+        syncUpdate(getTransient("RIPE-DBM-STARTUP-MNT") + "password:startup\noverride:       denis,override1")
```

Comment:
Also here, now using override

Reply:
A validation rule was added to prevent RIPE maintainers from being created unless override authentication is used.